packages/transactions/transactions-integration.test.js

```diff
@@ -0,0 +1,130 @@
+import {TestFixtures} from '@docknetwork/wallet-sdk-core/lib/fixtures';
+import {Wallet} from '@docknetwork/wallet-sdk-core/lib/modules/wallet';
+import {
+  mockTransaction,
+  waitFor,
+} from '@docknetwork/wallet-sdk-core/lib/test-utils';
+import {
+  Transactions,
+  AccountTransactions,
+  TransactionStatus,
+} from './lib/transactions';
+
+describe('Transactions integration test', () => {
+  let wallet: Wallet;
+  let account: Account;
+  let accountTx: AccountTransactions;
+  const txInput = {
+    amount: 1,
+    toAddress: TestFixtures.account2.address,
+  };
+
+  jest.setTimeout(60000);
+
+  beforeAll(async () => {
+    wallet = await Wallet.create();
+    account = await wallet.accounts.create(TestFixtures.account1);
+    accountTx = Transactions.with(account);
+  });
+
+  // it('Expect to get transaction fee', async () => {
```

Comment:
assuming these are commented out because the function above is commented out?

Reply:
fixed